### ISSUE TITLE
Better error message for improper "use strict"; placement in no-unused-expression

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3764]
### Bug Report

- __TSLint version__: 5.9.2
- __TypeScript version__: 2.8.*
- __Running TSLint via__: CLI & VS Code

#### JavaScript code being linted

```typescript
const fs = require("fs");

"use strict";
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "no-unused-expression": true
    }
}
```

#### Actual behavior

```
ERROR: 3:1  no-unused-expression  unused expression, expected an assignment or function call
```

#### Expected behavior

An error message saying the `"use strict";` should be before any other expressions. New users might not be familiar with it.

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
